src/main/docker/kafdrop.sh

```diff
@@ -40,7 +40,7 @@ if [ "$KAFKA_PROPERTIES" != "" ]; then
   echo Writing Kafka properties into $KAFKA_PROPERTIES_FILE
   echo "$KAFKA_PROPERTIES" | base64 --decode --ignore-garbage > $KAFKA_PROPERTIES_FILE
 else
-  rm $KAFKA_PROPERTIES_FILE |& > /dev/null | true
+  if [ -f $KAFKA_PROPERTIES_FILE ]; then touch $KAFKA_PROPERTIES_FILE; else rm -f $KAFKA_PROPERTIES_FILE; fi
```

Comment:
* Why are you touching the file if it exists?
* Shouldn't you add ``-r`` to the ``rm`` command? If it's not a regular file, it might be a directory.

Reply:
I've simplified this logic now to 

```bash
if [ ! -f $KAFKA_PROPERTIES_FILE ]; then touch $KAFKA_PROPERTIES_FILE; fi
```

The idea is that, if the default `kafka.properties` is being used, it should be written out as a blank file as Kafdrop will look for it at `KAFKA_PROPERTIES_FILE`.

Whereas, if it already exists (e.g. if the user has made it available through a volume mount), then just let it be.

Reply:
That pattern makes sense indeed. @ChristianCiach already [flagged the deletion long back](https://github.com/obsidiandynamics/kafdrop/commit/b07a84d745ffec682f32548ff1e24ab4dbabb8ed#r53695036), I just noticed.